### ISSUE TITLE
Call Out Beacon Action Not Working on Waypoints

### ISSUE THREAD

**Issue by JJGatchalian** [soundscape-community/soundscape#278]
When selecting the action to "Call Out Beacon" using the VoiceOver gesture of swiping up or down with one finger, when double tapping the Call Out Beacon action, it will call out the beacon and its distance on a single beacon but not for waypoints on a route. Double tapping Call Out Beacon on a waypoint does nothing. We should either make it work or remove the action, or at least have VoiceOver indicate that it doesn't work like this: "Call out Beacon, dimmed".
